test-e2e/step-definitions/index.ts

```diff
@@ -0,0 +1,13 @@
+import { Then, When } from '@cucumber/cucumber';
+import memory from '@qavajs/memory';
+import { Browser } from 'webdriverio';
+
+declare global {
+    var browser: Browser<'async'>;
+    var driver: Browser<'async'>;
+    var config: any;
+}
+
+Then('validate screenshot', function () {
```

Comment:
why do we need that step here?

Reply:
it's custom step to check captured screenshot, will be used later when I add e2e for memory steps

Reply:
got it, for now it looks redundant

Reply:
I'll remove it